rustls/src/x509.rs

```diff
@@ -1,5 +1,6 @@
 // Additional x509/asn1 functions to those provided in webpki/ring.
 
+#[allow(dead_code)] // if webpki is compiled out
```

Comment:
I think rather than indicating this with a comment we could use `cfg_attr` here:

```suggestion
#[cfg_attr(not(feature = "webpki"), allow(dead_code))]
```

Reply:
actually this is no longer needed since djc's other PR